Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:latest
```

Comment:
Have you considered using something like `alpine` instead?
Downloading a whole Ubuntu image for a couple of PDFs feels a bit overkill.

Reply:
I have, but that uses a different package manager, so the install script cannot be used with it. This PR used debian slim before, but I figured I might as well make this image as close to the github action as possible. If you think Debian slim is close enough to ubuntu, I can change it back.

Reply:
But actually, I didn't find a big difference in build times between the two, and you only need to build the image once and then you can keep using it, so I'd just leave Ubuntu as the base image

Reply:
Okay yeah then whatever. It's not about build times necessarily, more about how much space you're taking up on the host machine.

Reply:
Right. So the image with an Ubuntu base is 1.76GB. I just tried the Debian slim base image again, but it won't work, because of the newly-added package version locking for pandoc and weasyprint, since Debian and Ubuntu have slightly different versions of packages in their respective repos.